cypress/components/NotificationsDrawer.cy.tsx

```
@@ -36,21 +36,11 @@ const notificationDrawerData: NotificationData[] = [
-  {
-    resourceDefinitions: [],
-    permission: 'notifications:notifications:read',
-  },
-];
-
 const DrawerLayout = ({ markAll = false }: { markAll?: boolean }) => {
   const drawerPanelRef = useRef<HTMLDivElement>(null);
   const [isExpanded, setIsExpanded] = useState(false);
-  const { initialize, addNotification } = DrawerSingleton.Instance;
+  const { initialize, addNotification, setHasNotificationsPermissions } =
+    DrawerSingleton.Instance;
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)